ocs_ci/ocs/resources/objectconfigfile.py

```diff
@@ -135,3 +136,57 @@ def get(self, namespace=None):
         """
         out = self._run_command("get", namespace, out_yaml_format=True)
         return yaml.safe_load(out)
+
+    def describe(self, namespace=None):
+        """
+        Run ``oc describe`` on in this object file.
+
+        Args:
+            namespace (str): Name of the namespace where to deploy, overriding
+            self.project.namespace value (in a similar way how you can specify
+            any value to ``-n`` option of ``oc describe``.
+        """
+        out = self._run_command("describe", namespace, out_yaml_format=False)
+        return yaml.safe_load(out)
+
+    def wait_for_delete(self, resource_name="", timeout=60, sleep=3, namespace=None):
+        """
+        Wait for a resource to be deleted
+
+        Args:
+            resource_name (str): The name of the resource to wait
+                for (e.g.kube_obj_name)
+            timeout (int): Time in seconds to wait
+            sleep (int): Sampling time in seconds
+            namespace (str): Name of the namespace where to deploy, overriding
+                self.project.namespace value (in a similar way how you can specify
+                any value to ``-n`` option of ``oc get``.
+
+        Raises:
+            CommandFailed: If failed to verify the resource deletion
+            TimeoutError: If resource is not deleted within specified timeout
+
+        Returns:
+            bool: True in case resource deletion is successful
+
+        """
+
+        start_time = time.time()
+        while True:
+            try:
+                self.get(namespace=namespace)
+            except CommandFailed as ex:
+                if "NotFound" in str(ex):
+                    logger.info(f"{resource_name} got deleted successfully")
```

Comment:
I'd add a namespace to this print

Reply:
@ypersky1980  The intention of this validation is to check the respective kube_job deleted or not, and resource_name has it.
If the TC creates different namespace then for debugging or validation it will be difficult but we have one namespace but different kube_job objs created in that namespace, so felt specifying the kube_job obj would be more relevant. Let me know WDY.